images/linux/scripts/SoftwareReport/SoftwareReport.CachedTools.psm1

```diff
@@ -28,6 +28,27 @@ function Get-ToolcacheGoVersions {
     return Get-ChildItem $toolcachePath -Name | Sort-Object { [Version]$_ }
 }
 
+function Build-GoEnvironmentTable {
+    $variables = ,[PSCustomObject] @{
+         "Name" =  "GOROOT"
```

Comment:
Probably it makes sense to remove this variable from the report since you will remove it in https://github.com/actions/virtual-environments/pull/2668